docs/api/functions/host.md

```diff
@@ -29,14 +29,12 @@ An object representing the host which includes the host string, its parameters,
 ## Example
 
 ```ts
-import { createExternalRoutes, host } from '@kitbag/router'
-
-export const routes = createExternalRoutes([
-  {
-    name: 'home',
-    host: host('https://[subdomain].kitbag.dev', { subdomain: Number })
-  },
-])
+import { createExternalRoute, host } from '@kitbag/router'
+
+export const docs = createExternalRoute({
+  name: 'home',
+  host: host('https://[subdomain].kitbag.dev', { subdomain: Number })
```

Comment:
using `Number` here doesn't really make sense as an example. Probably fine to just use `String` even though that's the default